ios/RNPlayerViewManager.swift

```diff
@@ -74,6 +62,42 @@ public class RNPlayerViewManager: RCTViewManager {
         }
     }
 
+    func maybeCreateUserInterfaceConfig(
+        styleConfig: StyleConfig,
+        playerViewConfig: RNPlayerViewConfig?
+    ) -> UserInterfaceConfig? {
+#if os(iOS)
+        if styleConfig.userInterfaceType == .bitmovin {
+            let bitmovinUserInterfaceConfig = styleConfig
+                .userInterfaceConfig as? BitmovinUserInterfaceConfig ?? BitmovinUserInterfaceConfig()
+
+            if let uiConfig = playerViewConfig?.uiConfig {
+                bitmovinUserInterfaceConfig
+                    .playbackSpeedSelectionEnabled = uiConfig.playbackSpeedSelectionEnabled
+            }
+            bitmovinUserInterfaceConfig.hideFirstFrame = playerViewConfig?.hideFirstFrame ?? false
```

Comment:
In case we decide changing `hideFirstFrame` to optional.

```suggestion
            if let hideFirstFrame = playerViewConfig?.hideFirstFrame {
                bitmovinUserInterfaceConfig.hideFirstFrame = hideFirstFrame
            }
```